.eslintrc

```diff
@@ -17,9 +15,4 @@ globals:
   XPCOMUtils: false
```

Comment:
I'd be hard pressed to care any less... I think mostly depends on if you want to define once in one .eslitnrc (and have them leak everywhere), or maintain the globals in separate files.

If you prefer no globals in the eslintrc file, it's a quick fix for me to disable them and then silence the errors in the specific files.

I'm here to serve.